tangoObjects.py

```diff
@@ -9,13 +9,14 @@
 from builtins import str
 import redis
 import pickle
+from typing import Optional, Iterator, Tuple
 from queue import Queue
 from config import Config
 
 redisConnection = None
 
 
-def getRedisConnection():
+def getRedisConnection() -> redis.StrictRedis:
```

Comment:
Figuring out the type of this is tricky, it's actually `redis.client.Redis`. You can do it by installing [mypy](http://mypy-lang.org/), creating a test python file (say test.py), adding the following stub:

```
import redis
reveal_type(redis.StrictRedis(host='localhost', port=6379, db=0))
```

`reveal_type` is a feature from `mypy` that has an example here: https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html#when-you-re-puzzled-or-when-things-are-complicated